app/views/sessions/new.mobile.haml

```diff
@@ -2,47 +2,46 @@
 -#   licensed under the Affero General Public License version 3 or later.  See
 -#   the COPYRIGHT file.
 
-:css
-  .navbar-fixed-top{display: none;}
```

Comment:
I just had a quick look at the mobile login page and it looks like that element isn't there anymore so it would be ok to just remove this inline CSS. Whoever is going to merge this: it would be great if you could check again that there won't be a regression. :)